app/client/ui/GristTooltips.ts

```diff
@@ -234,6 +236,25 @@ see or edit which parts of your document.')
     dom('div', cssLink({href: commonUrls.helpAdminControls, target: "_blank"}, t('Learn more.'))),
     ...args,
   ),
+  uploadAttachments: (...args: DomElementArg[]) => cssTooltipContent(
+    cssMarkdownSpan(
+      t(
+        "This allows you to add attachments that are missing from external storage, e.g. in an imported document. " +
+        "Only .tar attachment archives downloaded from Grist can be uploaded here."
+      ),
+    ),
+    ...args,
+  ),
+  attachmentsNotIncluded: (...args: DomElementArg[]) => cssTooltipContent(
+    cssMarkdownSpan(
+      t(
+        "If you want to re-import this document, " +
+        "a separate attachments archive will be needed to restore attachments. "
+      ),
+      dom('div', cssLink({href: "", target: "_blank"}, t('Learn more.'))),
```

Comment:
Can you add a stub in common urls?

Reply:
I'll go a step further and delete this - this was actually dead code, I just realised. Will double check the rest of the PR too.